cpp/test/Glacier2/router/Client.cpp

```diff
@@ -484,6 +484,7 @@ CallbackClient::run(int argc, char** argv)
         try
         {
             baseC->ice_ping();
+            test(false);
```

Comment:
This important "test(false)" is unfortunately missing in all versions of this test.